x-pack/plugins/cloud_security_posture/public/pages/findings/use_findings_count.ts

```diff
@@ -23,40 +23,55 @@ interface FindingsAggs extends estypes.AggregationsMultiBucketAggregateBase {
   };
 }
 
-export const getFindingsCountAggQuery = ({ index, query }: FindingsBaseEsQuery) => ({
-  index,
+interface UseFindingsCounterData {
+  passed: number;
+  failed: number;
+  newPitId: string;
+}
+
+export const getFindingsCountAggQuery = ({
+  query,
+  pitIdRef,
+}: Omit<FindingsBaseEsQuery, 'setPitId'>) => ({
   size: 0,
   track_total_hits: true,
   body: {
     query,
     aggs: { count: { terms: { field: 'result.evaluation.keyword' } } },
+    pit: { id: pitIdRef.current },
```

Comment:
IMO we should pass the actual string and not the reference. 

by passing the reference we're assuming the value we expect will be there when we read. even if this is working now, it's probably safer to pass the actual string.